cli/deno_error.rs

```diff
@@ -1,6 +1,18 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
-use crate::diagnostics::Diagnostic;
-use crate::fmt_errors::JSError;
+
+/// This module implements error serialization; it
+/// allows to serialize Rust errors to be sent to JS runtime.
+///
+/// Currently it is deeply intertwined with `ErrBox` which is
+/// not optimal since not every ErrBox can be "JS runtime error";
+/// eg. there's no way to throw JSError/Diagnostic from within JS runtime
+///
+/// TODO:
+///  - rename ErrorKind::Other to WindowError/GenericError
+///  - remove ErrorKind::WouldBlock
+///  - possibly rename "GetErrorKind" to "RuntimeError" - it will allow
+///    better semantic separation of errors which can be sent to JS runtime
+///    eg. each RuntimeError is ErrBox, but not every ErrBox is RuntimeError
 use crate::import_map::ImportMapError;
 pub use crate::msg::ErrorKind;
```

Comment:
I think ErrorKind should be moved this module. It's very related to DenoError.

Reply:
Yeah, `msg.rs` is cruft